src/Components/Projects/Projects.tsx

```diff
@@ -122,7 +114,6 @@ const Projects: React.FC = () => {
         getAllProjectsList();
       })
       .catch((error) => {
-        setIsLoading(false);
```

Comment:
You  have just forgot here to add the finally() block and set the loading state back to false

Reply:
done